src/msix/msix.cpp

```diff
@@ -348,18 +348,6 @@ MSIX_API HRESULT STDMETHODCALLTYPE PackBundle(
     }
     //TODO:: Error if directoryPath is a file

```

Comment:
>//TODO:: Error if directoryPath is a file [](start = 4, length = 41)

Remove

Reply:
I am keeping this for now, to remember the TODO. I need to add this portion to the code.